modules/betweenBidAdapter.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+Module Name: Example Bidder Adapter
```

Comment:
Please update the template and add actual contact info.

Reply:
done